### PR TITLE
[DOC] Add ./ansible.cfg to docs where missing

### PR DESCRIPTION
**Pull request by Klaas-**
##### SUMMARY

./ansible.cfg is missing in manpage of ansible, I added it and all other occurrences of the config files I found. 

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

core

##### ANSIBLE VERSION

```
ansible 2.4.3.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/klaas/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.14 (default, Feb 17 2018, 10:42:17) [GCC 7.3.1 20180130 (Red Hat 7.3.1-2)]
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Review by dharmabumstead — APPROVED** [review]
Doc changes OK. Needs core team review and signoff.

**Comment by Klaas-** [comment]
So this seems like https://access.redhat.com/errata/RHSA-2018:2166 // https://github.com/ansible/ansible/pull/42070 makes this obsolete?

**Comment by Klaas-** [comment]
And I take full credit for finding this 5 months ago :D

**Comment by Klaas-** [comment]
@bcoca :)

**Pull request closed without merging**
